aries-site/src/pages/foundation/color.mdx

```diff
@@ -0,0 +1,260 @@
+import { Button } from 'grommet';
+import { Figma } from 'grommet-icons';
+
+import { ColorSwatch } from '../../components';
+import {
+  ButtonRow
+} from '../../layouts';
+import { 
+  BackgroundDark,
+  BackgroundLight,
+  BordersDark,
+  BordersLight,
+  BrandColor, 
+  ContrastDark,
+  ContrastLight,
+  CorePalette, 
+  DarkModeContrast,
+  DarkPalette,
+  ElevationDark,
+  ElevationLight,
+  FocusColor,
+  GraphColors,
+  InputsDark,
+  InputsLight,
+  LightPalette,
+  LightModeContrast,
+  Overlay,
+  StatusDark,
+  StatusLight,
+  TextDark,
+  TextLight
+} from '../../examples';
+
+You will notice that the HPE Design System Color guidance is
```

Comment:
Don't think it matters in terms of how it renders, but noticing this has line breaks within paragraphs in the mdx as opposed to other files that just have on long line. Do we want to make things one line, or was this intentional for readability?

Reply:
I find it easier for readability. The long lines tend to bother me, but just a preference.